examples/embed.py

```diff
@@ -11,30 +11,56 @@
 ), "Your API key should be stored in an environment variable named API_KEY."
 
 with TwelveLabs(API_KEY) as client:
+    embed_tasks = client.embed.task.list()
+    for task in embed_tasks:
+        print(
+            f"Embedding task: id={task.id} status={task.status} created_at={task.created_at}"
+        )
+        if task.metadata is not None:
+            print(f"  metadata: {task.metadata}")
+
+    def print_video_embeddings(id: str):
+        task = client.embed.task.retrieve(id)
+        if task.video_embeddings is not None:
+            for v in task.video_embeddings:
+                print(
+                    f"embedding_scope={v.embedding_scope} start_offset_sec={v.start_offset_sec} end_offset_sec={v.end_offset_sec}"
+                )
+                print(f"embeddings: {", ".join(str(v.embedding.values))}")
+
+    def on_task_update(task: EmbeddingsTask):
+        print(f"  Status={task.status}")
+
     engine_name = "Marengo-retrieval-2.6"
-    embedding = client.embed.create(
+
+    text_embedding = client.embed.create(
         engine_name=engine_name,
         text="man walking across the street",
         text_truncate="start",
     )
-    print(f"Created embedding: engine_name={embedding.engine_name}")
+    print(f"Created text embedding: engine_name={text_embedding.engine_name}")
```

Comment:
I think this should print the embedding itself.

Reply:
that'd be great :)